build.gradle

```diff
@@ -106,17 +106,9 @@ allprojects {
         }
     }
 
-    plugins.withId('com.palantir.baseline-error-prone', {
-        tasks.withType(JavaCompile).configureEach {
-            options.errorprone.disableWarningsInGeneratedCode = true
-            options.errorprone.errorproneArgs.addAll([
-                '-XepDisableWarningsInGeneratedCode',
-                '-XepExcludedPaths:.*(generated).*',
-            ])
-
-            // temporarily relax constraints until we can fix all violations
-            Set<String> errorsToRelax = [
-                'CatchBlockLogException',
+    tasks.withType(JavaCompile) {
+        // temporarily relax constraints until we can fix all violations
+        options.errorprone.disable 'CatchBlockLogException',
```

Comment:
Do we actually want to disable? I tend to grep the `(error|warning):` from build log to fix these up

Reply:
We can set them to warn instead, no strong preference. I've normally added things to a list in order to burn through it later since it's easier to promote a single check and fix all occurrences.

Mind if I push a few automated check fixes to cut down on the warnings along with this?